pyproject.toml

```diff
@@ -78,6 +79,7 @@ namespaces = false
 
 
 [tool.pytest.ini_options]
+testpaths = ["tests/pytest"]
```

Comment:
This is used to call pytest from root dir and avoid the warning about `conftest.py`